index.js

```diff
@@ -178,4 +178,5 @@ module.exports = build
 module.exports.build = build
 module.exports.prettyFactory = prettyFactory
 module.exports.colorizerFactory = colors
+module.exports.isColorSupported = isColorSupported
```

Comment:
Can you mention this in documentation?

Reply:
hey @kibertoad just added a small section to inform the users about this new option. Let me know if seems good?

Reply:
thank you, it's great!